frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -54,7 +55,9 @@ const TranslationPage = () => {
   const [changedStoryLines, setChangedStoryLines] = useState<
     Map<number, StoryLine>
   >(new Map());
-  const [percentageComplete] = useState(25);
+  const [numTranslatedLines, setNumTranslatedLines] = useState(0);
+  const [numStoryLines, setNumStoryLines] = useState(0);
+  const [percentageComplete, setPercentageComplete] = useState(0);
```

Comment:
Does percentageComplete need to be a state variable? It won't trigger any changes not already found by the `numTranslatedLines`, and can be calculated as a const or even in line, I think